packages/appstore/src/components/modals/account-type-modal/account-type-modal.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { Button, Modal, DesktopWrapper, MobileDialog, MobileWrapper, UILoader, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { observer } from 'mobx-react-lite';
+import classNames from 'classnames';
+import { useStores } from 'Stores/index';
+import TradigPlatformIconProps from 'Assets/svgs/trading-platform';
+
+type TModalContent = {
+    account_type_card: string;
+    selectAccountTypeCard: React.Dispatch<React.SetStateAction<string>>;
+};
+
+const ModalContent = ({ account_type_card, selectAccountTypeCard }: TModalContent) => {
+    const cardSelection = (cardType: string) => {
+        selectAccountTypeCard(account_type_card === cardType ? '' : cardType);
+    };
+    return (
+        <div className='account-type-card__wrapper'>
+            <div
+                className={classNames('account-type-card', {
+                    'account-type-card--selected': account_type_card === 'Derived',
+                })}
+                onClick={() => cardSelection('Derived')}
```

Comment:
better to define 'Derived' and 'Financial' types as an object to prevent hardcoding

Reply:
used values of the created array